packages/web3-eth-accounts/src/types.ts

```diff
@@ -0,0 +1,25 @@
+import { FeeMarketEIP1559TxData, AccessListEIP2930TxData, TxData } from '@ethereumjs/tx';
+
+export type signatureObject = {
+	messageHash: string;
+	r: string;
+	s: string;
+	v: string;
+};
+
+export type signTransactionResult = signatureObject & {
+	rawTransaction: string;
+	transactionHash: string;
+};
+
+export type signTransactionFunction = (
+	transaction: TxData | AccessListEIP2930TxData | FeeMarketEIP1559TxData,
+	privateKey: string,
+) => signTransactionResult;
```

Comment:
```suggestion
```
 I don't think this is needed, TypeScript will infer the method signature

Reply:
as we are returning this function in return type of other functions so good to mention type for user's usage or visibility